### ISSUE TITLE
getRange().reduce

### ISSUE THREAD

**Issue by angrymouse** [kriszyp/lmdb-js#246]
Would be cool to implement lazy reduce function on getRange() - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce

**Comment by kriszyp** [comment]
Yes, I agree, that would be a nice addition, will try to do it at some point.
